docker-imposm3/Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM golang:1.10
+FROM golang:buster
```

Comment:
please use a fixed version number instead

Reply:
base image changed in the last commit